src/frontend/src/components/Link/LinksEditView.tsx

```diff
@@ -13,7 +13,7 @@ const LinksEditView: React.FC<{
   ls: FieldArrayWithId[];
   register: UseFormRegister<ProjectFormInput>;
   watch: UseFormWatch<ProjectFormInput>;
-  append: UseFieldArrayAppend<any, any>;
+  append: UseFieldArrayAppend<ProjectFormInput, 'links'>;
```

Comment:
can you also move this file to `src\frontend\src\pages\ProjectDetailPage\ProjectForm` it's pretty tightly tied to that form and not super reuseable in other places as it is rn, so it should be in that folder